src/js/va-letters/va-letters-entry.jsx

```diff
@@ -0,0 +1,36 @@
+import 'core-js';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createHistory } from 'history';
+import { IndexRedirect, Route, Router, useRouterHistory } from 'react-router';
+import { Provider } from 'react-redux';
+
+import VALettersApp from './containers/VALettersApp.jsx';
+import initReact from '../common/init-react';
+import { commonStore } from '../common/store';
+import routes from './routes.jsx';
+
+require('../common');  // Bring in the common javascript.
+require('../../sass/va-letters.scss');
+require('../login/login-entry.jsx');
+
+
+const history = useRouterHistory(createHistory)({
+  basename: '/va-letters'
+});
+
+function init() {
+  ReactDOM.render((
+    <Provider store={commonStore}>
+      <Router history={history}>
+        <Route path="/" component={VALettersApp}>
+          <IndexRedirect to="/confirm-address"/>
+          {routes}
```

Comment:
Hm that's odd, it was working for me locally. Maybe try again after running `npm run build` and restarting the `watch` task?